### DISCUSSION TITLE
Strapi Admin Dashboard Throws Error in Production only

### DISCUSSION THREAD

**Discussion by ktayah** [strapi/strapi#6156] [Legacy Topics]
I just got a Strapi server up and running on my domain on an Ubuntu EC2 Instance. All is well however, I can no longer get access to the dashboard after I login. Visiting the admin page correctly shows me the Login/Signup prompt however after I am prompted with a white screen. Here is the error thrown to the console.

![image](https://user-images.githubusercontent.com/32495802/81509789-25545b80-92db-11ea-8604-3008686f08bf.png)

**Comment by derrickmehaffy** [comment]
What Strapi version is your project, and generally this is caused by an old build of the admin (you have to rebuild the admin after any update to package versions.

Sometimes it's possible your `yarn.lock` or `package-lock.json` is caching some older package version as well.

**Comment by lauriejim** [comment]
Hello, please follow the issue template [here](https://github.com/strapi/strapi/blob/master/.github/ISSUE_TEMPLATE/BUG_REPORT.md).

This will help us to understand the WHY of your issue.
